src/containers/ProfileDetails/components/Radius/components/ManagementSubnet/index.js

```diff
@@ -17,6 +17,7 @@ const ipPattern = /\b\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}\b/;
 
 const ManagementSubnetModal = ({ onSuccess, onCancel, visible, title, subnet }) => {
   const [form] = Form.useForm();
+  const [cidrPrefix, setCidrPrefix] = useState();
```

Comment:
remove state, just do the conversion on form submit and pass the value to onSuccess